src/ui/units/dash/components/DashActionPanel/DashActionPanel.tsx

```diff
@@ -81,9 +78,15 @@ export type ActionPanelProps = OwnProps & StateProps & DispatchProps;
 type ActionPanelState = {};
 
 class DashActionPanel extends React.PureComponent<ActionPanelProps, ActionPanelState> {
+    isFakeEntry = Boolean(
```

Comment:
Problem case:
- open new dash
- open empty created dash
Expect: updated actionPanel in top
Reality: old actioPanel

Reply:
Getter solved the problem